packages/cashier/src/containers/cashier/cashier.tsx

```diff
@@ -170,7 +166,6 @@ const Cashier = observer(({ history, location, routes: routes_config }: TCashier
 
     return (
         <FadeWrapper is_visible={is_visible} className='cashier__page-wrapper' keyname='cashier__page-wrapper'>
-            <AccountPromptDialog />
```

Comment:
Rendering the dialog component where it's needed instead of the root of the app.